app/controllers/organisations_controller.rb

```diff
@@ -27,15 +27,15 @@ def index
   # GET /organisations/1
   # GET /organisations/1.json
   def show
-    @organisation = Organisation.find(params[:id])
+    organisations = Organisation.where(id: params[:id])
+    @organisation = organisations.first!
     @pending_org_admin = current_user.pending_org_admin? @organisation if current_user
     @editable = current_user.can_edit?(@organisation) if current_user
     @deletable = current_user.can_delete?(@organisation) if current_user
     @can_create_volunteer_op = current_user.can_create_volunteer_ops?(@organisation) if current_user
     @grabbable = current_user ? current_user.can_request_org_admin?(@organisation) : true
     @can_propose_edits = current_user.present? && !@editable
-   # @next_path = current_user ? organisation_user_path(@organisation.id, current_user.id) : new_user_session_path
-    @markers = build_map_markers([@organisation])
+    @markers = build_map_markers(organisations)
```

Comment:
can we not just get the first organisation with find, and then pass [@organisation] to build_map_markers?